extensions-contrib/materialized-view-selection/src/main/java/org/apache/druid/query/materializedview/DerivativeDataSourceManager.java

```diff
@@ -219,33 +219,36 @@ private void updateDerivatives()
   private long getAvgSizePerGranularity(String datasource)
   {
     return connector.retryWithHandle(
-        new HandleCallback<Long>() {
+        new HandleCallback<>()
+        {
           Set<Interval> intervals = new HashSet<>();
           long totalSize = 0;
+
           @Override
           public Long withHandle(Handle handle)
           {
             handle.createQuery(
-                StringUtils.format("SELECT start,%1$send%1$s,payload FROM %2$s WHERE used = true AND dataSource = :dataSource",
```

Comment:
Since the PR already touches too many files, maybe avoid the formatting changes in this PR and keep it limited to only using the diamond operator.